libcontainer/cgroups/fscommon/fscommon.go

```diff
@@ -39,6 +40,14 @@ func ReadFile(dir, file string) (string, error) {
 	return buf.String(), err
 }
 
+// FileExists returns true if a file exists in dir, otherwise false.
+func FileExists(dir, file string) bool {
+	if _, err := os.Stat(path.Join(dir, file)); err == nil {
+		return true
+	}
+	return false
```

Comment:
Should return true if !errors.Is(err, os.ErrNotExist)

Reply:
Thanks, definitely so! Fixed.